packages/editor-ui/src/Interface.ts

```diff
@@ -856,8 +856,19 @@ export interface INodeMetadata {
 	parametersLastUpdatedAt?: number;
 }
 
-export interface IRootState {
+export interface IExecutionsState {
 	activeExecutions: IExecutionsCurrentSummaryExtended[];
+	executionId: string | null;
+	executingNode: string | null;
+	executionWaitingForWebhook: boolean;
```

Comment:
these three flags are more related to currently active workflow than current execution.. executions store should contain all historically executed workflows and any data related tot that.. these are more related to current active workflow than executions..